server/config.py

```diff
@@ -1,3 +1,31 @@
 import os
+from dotenv import load_dotenv
+basedir = os.path.abspath(os.path.dirname(__file__))
+load_dotenv(".env")
 
 TEAM_NAME = os.environ['TEAM_NAME']
+
+class Config(object):
+    DEBUG = False
+    TESTING = False
+    CSRF_ENABLED = True
+    SECRET_KEY = 'this-really-needs-to-be-changed'
```

Comment:
get this from the os environ

Reply:
Redirected to os environ. Please review at .env.example whether it is proper or not.